modules/sample-modules/_index.md

```diff
@@ -1,10 +1,46 @@
 ---
-title: Sample modules
+title: Sample modules and how to's
 weight: 80
 ---
 
-# Sample modules
+# Sample modules and how to guides
+
+The PrestaShop Community made some example modules and how to guides to help you understand how to implement hooks, admin grids, manage entities... and a lot more.
 
 {{% children %}} 
 
+## Our example modules repository
+
 * [See all our example modules (GitHub)](https://github.com/PrestaShop/example-modules)
+
+This repository hosts example modules built and maintained by PrestaShop.
+These modules demonstrate useful usecases for developers willing to customize the software.
+
+### Extending Symfony pages
+
+- [Demo Extending a Symfony Form - 1](https://github.com/PrestaShop/example-modules/tree/master/demoextendsymfonyform1) - how to insert an input inside a Symfony form
```

Comment:
Consider putting the right text inside the anchor, it's better for SEO. Consider the search terms a person would use when googling for this information.
```suggestion
- [Extending a Symfony form - how to insert a field inside an existing form](https://github.com/PrestaShop/example-modules/tree/master/demoextendsymfonyform1)
```